content/en/account_management/cloud_provider_authentication.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Cloud-based Authentication
+aliases:
+    - /account_management/cloud_authentication/
+algolia:
+  tags: ['cloud authentication', 'aws authentication', 'terraform provider']
+further_reading:
+- link: "/getting_started/integrations/terraform/"
+  tag: "Documentation"
+  text: "Managing Datadog with Terraform"
+- link: "/account_management/api-app-keys/"
+  tag: "Documentation"
+  text: "API and Application Keys"
+- link: "/integrations/amazon_web_services/"
+  tag: "Documentation"
+  text: "AWS Integration"
+---
+
+{{< callout url="https://www.datadoghq.com/product-preview/cloud-provider-authentication/" btn_hidden="false" header="Join the Preview!" >}}
+Cloud-based authentication is in Preview. Complete the form to request access.
+{{< /callout >}}
+
+## Overview
+
+Cloud-based authentication lets you authenticate the Datadog Terraform provider using cloud credentials instead of static API and application keys.
+
+The preview only supports **AWS**, with other cloud providers planned for future releases.
+
+## AWS authentication process
+
+The authentication process uses the [AWS Security Token Service (STS)][1] to verify your identity:
+
+1. **Proof generation:** The Datadog Terraform provider creates a signed AWS STS `GetCallerIdentity` request using your current AWS credentials
+2. **Proof validation:** Datadog validates the proof by calling AWS STS, which returns your AWS ARN, user ID, and account ID
+3. **Identity mapping:** Your AWS identity is mapped to a Datadog service account or user account based on your organization's configuration
+4. **Token issue:** If validation succeeds, Datadog issues a temporary JWT token for API access
+5. **API authentication:** The token is used for subsequent Datadog API calls
+
+**Note:** Prefer mapping ARNs to a Datadog service account.
```

Comment:
What is better about mapping ARNs to a service account? Is the alternative mapping ARNs to a user account? Also, should this line be moved closer to line 49, when you talk about mapping ARNs?

Reply:
Yes, removing the API reference links sounds good.